From dc1d1bb511e1a0afbee2d79c39d73de0dc1f64dd Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Tue, 18 Jun 2019 10:32:11 -0500
Subject: [PATCH 1/2] build

--
rebased for update to 3.32.1

Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>
---
 meson.build | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/meson.build b/meson.build
index b7c7731..221def2 100644
--- a/meson.build
+++ b/meson.build
@@ -146,6 +146,7 @@ endif
 tracker_sparql = dependency('tracker-sparql-2.0')
 x11 = dependency('x11')
 xml = dependency('libxml-2.0', version: '>= 2.7.8')
+fontconfig = dependency('fontconfig', required: false)
 
 fontconfig = dependency('fontconfig', required: false)
 
@@ -173,6 +174,12 @@ else
   fontconfig_cache_path = join_paths(libdir, 'fontconfig/cache')
 endif
 
+if fontconfig.found()
+   fontconfig_cache_path = fontconfig.get_pkgconfig_variable('cachedir')
+else
+   fontconfig_cache_path = join_paths(libdir, 'fontconfig/cache')
+endif
+
 conf.set_quoted('APPLICATION_ID', application_id)
 conf.set_quoted('GETTEXT_PACKAGE', 'nautilus')
 conf.set_quoted('INSTALL_PREFIX', prefix)
-- 
2.22.0

