From d0f20de1b9490487facca1e953c11506e3dccf33 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Wed, 13 Sep 2017 14:49:29 -0500
Subject: [PATCH] autostart desktop file goes to /usr/share/xdg and not
 sysconfdir

---
 data/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index a679997..18422c4 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -26,7 +26,7 @@ desktop_autostart = i18n.merge_file ('desktop-autostart',
                                      input: 'nautilus-autostart.desktop.in',
                                      output: 'nautilus-autostart.desktop',
                                      install: true,
-                                     install_dir: join_paths (sysconfdir, 'xdg', 'autostart'),
+                                     install_dir: join_paths (datadir, 'xdg', 'autostart'),
                                      po_dir: po_dir,
                                      type: 'desktop')
 
-- 
2.14.1

